packages/sdk/src/swap/sdk.ts

```diff
@@ -218,4 +223,16 @@ export class SwapSDK {
 
     return { minimumSwapAmounts: minimumDepositAmounts, maximumSwapAmounts };
   }
+
+  async getRequiredBlockConfirmations(
```

Comment:
i think i would return a `ChainMap` here because we return a map from `getSwapLimits` too. makes sense to keep it consistent if there is no reason against it